apps/portal/components/pagesComponents/profileScreen/ChangeEmailModal.tsx

```
@@ -68,11 +70,11 @@ const ChangeEmailModal: React.FC<ICreateFolderModalProps> = ({
         />
       }
     >
-      <h2 className={styles.modalHeader}>Email Address</h2>
+      <h2 className={styles.modalHeader}>{t('page.profile.emailAddress')}</h2>
 
-      <label>Current email address:</label>
+      <label>{t('page.profile.currentEmail')}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)